.teamcity/Gradle_Check/configurations/StagePasses.kt

```diff
@@ -123,6 +123,9 @@ class StagePasses(model: CIBuildModel, stage: Stage, prevStage: Stage?, contains
                     if (subProject.containsSlowTests && stage.omitsSlowProjects) {
                         return@forEach
                     }
+                    if (subProject.hasOnlyUnitTests()) {
```

Comment:
Seems like this can be extracted as one method 

```
func shouldSkip() {
    return shouldBeSkipped(subProject, testCoverage) || (subProject.containsSlowTests && stage.omitsSlowProjects) || subProject.hasOnlyUnitTests();
}
```

So that this method can be shared with `FunctionalTestProject` and `CIConfigIntegrationTests.kt`?

Reply:
What good is the test if it doesn't assert that the method does the same thing? I think those two logics should be separate.

I refactored the production code, so it is hopefully easier to understand.